src/main/kotlin/org/rust/ide/inspections/fixes/RenameFix.kt

```diff
@@ -34,4 +36,7 @@ class RenameFix(
         }, {
             RefactoringFactory.getInstance(project).createRename(it, newName).run()
         })
+
+    override fun generatePreview(project: Project, previewDescriptor: ProblemDescriptor): IntentionPreviewInfo =
```

Comment:
It would be nice to support preview here eventually, at least for local changes like variable renaming

Reply:
https://github.com/intellij-rust/intellij-rust/pull/9699#discussion_r1018439359